mocks/router/middlewares/JSONHandler.go

```diff
@@ -28,7 +32,6 @@ func (_m *JSONHandler) Execute(w http.ResponseWriter, r *http.Request) (interfac
 		}
 	}
 
-	var r1 httperrors.HTTPError
```

Comment:
Remove this changes from commit "fix typo in Authentication" because it's not a typo.